changeFeedSamples/trackingChangesToBlobs/src/main/java/exampleEventCreator/ExampleEventCreator.java

```diff
@@ -0,0 +1,85 @@
+package exampleEventCreator;
+
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.azure.storage.blob.specialized.BlockBlobClient;
+import com.azure.storage.common.StorageSharedKeyCredential;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class ExampleEventCreator {
+    /**
+     * Sets up example events to test changefeed with
+     */
+    public static void main(String[] args) throws IOException {
+        // Delay between runs of adding new events
+        int interval = 3600000;
+
+        Path currentPath = Paths.get(System.getProperty("user.dir"));
+        Path pathToDir = Paths.get(currentPath.toString(), "changeFeedSamples",
+                "trackingChangesToBlobs", "src", "main", "java", "exampleEventCreator");
+        String configPath = Paths.get(pathToDir.toString(), "app.config").toString();
+
+        // Extracting variables from config file
+        InputStream input = new FileInputStream(configPath);
+        Properties prop = new Properties();
+        prop.load(input);
+        String sharedKeyCred = prop.getProperty("sharedKeyCred");
+        String storageAccount = prop.getProperty("storageAccount");
+        
+        // Create a Timer for creating events on a consistent interval
+        Timer timer = new Timer();
+        TimerTask task = new ExampleEventCreatorHelper(storageAccount, sharedKeyCred);
+
+        // Running on schedule
+        timer.scheduleAtFixedRate(task, 0, interval);
+    }
+}
+
+class ExampleEventCreatorHelper extends TimerTask {
+    String storageAccount;
+    String sharedKeyCred;
+    int blobCount = 0;
+
+    public ExampleEventCreatorHelper(String storageAccount, String sharedKeyCred) {
+        this.storageAccount = storageAccount;
+        this.sharedKeyCred = sharedKeyCred;
+    }
+
+    // Creates a new blob every time timer is triggered with file name "#exampleBlob.txt" where # is an int.
+    public void run() {
+        String storageAccountUrl = "https://" + storageAccount + ".blob.core.windows.net";
+        String containerName = "test-changefeed-container";
+        String blobName = "exampleBlob.txt";
```

Comment:
Should these be placed in the config?

Reply:
On the other hand, the user may want to filter by something else so I think its fine to have them in code

Reply:
We discussed this, and decided that since we already require the user to go into the code to adjust the filters, if they wanted to change the container and blob name they would already be looking at the code and can change it easily